modules/aws/ecs.go

```diff
@@ -90,6 +90,56 @@ func DeleteEcsClusterE(t *testing.T, region string, cluster *ecs.Cluster) error
 	return err
 }
 
+// GetEcsService fetches information about specified ECS service.
+func GetEcsService(t *testing.T, region, clusterName, serviceName string) *ecs.Service {
+	service, err := GetEcsServiceE(t, region, clusterName, serviceName)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I know this file doesn't have this pattern yet, but we've started shortening this to a one liner: `require.NoError(t, err)`. Can you apply to these functions?

Reply:
Done.